app/views/compute_resources_vms/form/vmware/_network.html.erb

```diff
@@ -2,11 +2,7 @@
   :class => "col-md-3 vmware_type",
   :label => _('NIC type'), :label_size => "col-md-3", :disabled => !new_vm
 %>
-<% if new_vm %>
-  <%= select_f f, :network, vsphere_networks(compute_resource), :first, :last, { },
-    :class => "col-md-3 vmware_network",
-    :label => _('Network'), :label_size => "col-md-3"
-  %>
-<% else %>
-  <%= text_f f, :network, :class => 'col-md-3 vmware_network', :label => _("Network"), :label_size => "col-md-3", :readonly => true %>
-<% end %>
+<%= select_f f, :network, vsphere_networks(compute_resource), :first, :last, { },
+  :class => "col-md-3 vmware_network",
+  :label => _('Network'), :label_size => "col-md-3", :disabled => !new_vm
```

Comment:
This is not disabled properly when the VM already exists. We currently don't support updating this so the select field should be disabled.

Reply:
uups, you are right, but is not presented by this PR. I will work on fix and make separate PR, ok?

Reply:
Can we do it as part of this PR? It should be trivial to fix and I believe the input field was disabled. So I do consider this a regression.

Reply:
Done. You were right, just a typo in new_vm guessing so pointless to make new PR.